units/container-getty@.service.in

```diff
@@ -25,10 +25,9 @@ Conflicts=rescue.service
 Before=rescue.service
 
 [Service]
-# The '-o' option value tells agetty to replace 'login' arguments with an
-# option to preserve environment (-p), followed by '--' for safety, and then
-# the entered username.
-ExecStart=-/sbin/agetty -o '-p -- \\u' --noclear --keep-baud - 115200,38400,9600 $TERM
+# The '-o' option value tells agetty to replace 'login' arguments with an option to preserve environment (-p),
+# followed by '--' for safety, and then the entered username.
+ExecStart=-/sbin/agetty -o '-p -- \\u' - $TERM
```

Comment:
why drop `--noclear`?

Reply:
It didn't seem to do anything, I didn't notice a difference in behavior with or without it.

Reply:
hmm, i think clearing the screen is an ANSI sequence, not a tty call, iirc.

Reply:
Added it back